extensions/stackdriver/stackdriver.cc

```diff
@@ -432,6 +456,20 @@ bool StackdriverRootContext::configure(size_t configuration_size) {
     tcp_log_entry_timeout_ = getTcpLogEntryTimeoutNanoseconds();
   }
 
+  // Extract metric tags expressions
+  cleanupMetricsExpressions();
+  for (const auto& override : getMetricsTagOverrides(config_)) {
+    uint32_t token;
+    auto metric = std::get<0>(override);
```

Comment:
Why do we have this intermediate tuple vector data structure, instead of directly constructing the expression and push it to `metrics_expressions_`? We can make `getMetricsTagOverrides()` take `metrics_expressions_` as input and build everything there.

Reply:
yeah, some of this is an artifact of just trying to piece together a set of working code. there is a definite need for cleanup and streamlining, but I wanted to make sure the broad strokes looked ok first.